src/pages/Signup.jsx

```diff
@@ -22,6 +22,7 @@ const SignupScreen = () => {
   const [username, setUsername] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [Confirmpassword, setConfirmPassword] = useState("");
```

Comment:
use camelCase please, instead of Confirmpassword use confirmPassword. It's a convention